src/client.ts

```diff
@@ -0,0 +1,94 @@
+import { workspace, WorkspaceFolder, Uri, window, OutputChannel } from "vscode";
+
+import {
+  LanguageClient,
+  LanguageClientOptions,
+  ServerOptions,
+  TextDocumentFilter,
+} from "vscode-languageclient/node";
```

Comment:
This should get receive more attention as part of
Abstract node specific imports #36